shop/settings.py

```diff
@@ -123,4 +123,6 @@
 STATICFILES_DIRS=[os.path.join(BASE_DIR,'static')]
 CRISPY_TEMPLATE_PACK='bootstrap4'
 LOGIN_REDIRECT_URL='user_home'
-LOGIN_URL='login'
\ No newline at end of file
+LOGIN_URL='login'
+MEDIA_ROOT=os.path.join(BASE_DIR,'media')
+MEDIA_URL='/media/'
```

Comment:
This MEDIA_ROOT explains where in our project that our media (images and videos will be uploaded). The MEDIA_URL explains where on our project the exact folder in which the media will be stored.